WordPress/src/main/java/org/wordpress/android/WordPress.java

```diff
@@ -156,6 +156,13 @@ public void onCreate() {
         HelpshiftHelper.init(this);
         initWpDb();
 
+        // EventBus setup
+        EventBus.TAG = "WordPressEvent";
```

Comment:
use the `WordPress-EVENT` string tag to match our log format (defined here https://github.com/wordpress-mobile/WordPress-Android/blob/issue/2331-reader-event-bus/libs/utils/WordPressUtils/src/main/java/org/wordpress/android/util/AppLog.java#L37-37)